### PR TITLE
Fix: deletion race condition

### PR DESCRIPTION
**Pull request by Meldiron**

### CODE REVIEW

File: app/http.php [hunk 1]
```diff
@@ -765,11 +770,6 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
                 'duration' => $executionTime + $coldStartTime,
             ];
 
-            // Update swoole table
-            $runtime = $activeRuntimes->get($activeRuntimeId);
-            $runtime['updated'] = \time();
-            $activeRuntimes->set($activeRuntimeId, $runtime);
-
```
Comment (christyjacob4) [inline app/http.php:772]: I don't think you should remove this.. This can be there as well 🆗

Comment (christyjacob4) [reply app/http.php:772]: what about also updating the `updated` time after the loop for runtime creation ?

Comment (Meldiron) [reply app/http.php:772]: I moved code to after runtime creation. In case of no cold-start, this runs instantly.

File: app/http.php [hunk 2]
```diff
@@ -634,6 +634,11 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
                 }
             }
 
+            // Update swoole table
+            $runtime = $activeRuntimes->get($activeRuntimeId);
```
Comment (christyjacob4) [inline app/http.php:638]: This should be done before the createRuntimeRequest.

The runtime server can still get deleted by the maintenance worker

File: app/http.php (hunk 1 above)
Comment (Meldiron) [reply app/http.php]: I put it back.

File: app/http.php (hunk 2 above)
Comment (Meldiron) [reply app/http.php]: Moved above createRuntimeRequest

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**